content/talks/resilient-systems.rst

```diff
@@ -0,0 +1,117 @@
+public: yes
+speakers: [miriam]
+headline:
+  - type: Training & Consulting
+    tagline: |
+      Upgrade your app with resilient systems training, tooling & consulting
+image:
+  - src: 'talks/miriam.jpg'
+event_type: workshop
+grid_template: true
+brag: |
+  Upgrade your app with resilient systems
+  training, tooling & consulting
+modules:
+  - title: Resilient & Universal Code
+    summary: |
+      Browser & device support
+      doesn't have to be difficult and expensive.
+      We'll cover a number of techniques
+      for building more resilient and universal
+      front end code,
+      while taking advantage of new browser features
+      as they become available.
+  - title: Intrinsic & Responsive Layouts
+    summary: |
+      We'll cover the web layout system as a whole --
+      grids, flow, flexbox, alignment,
+      intrinsic sizing, and queries --
+      showing when and how to mix and match techniques
+      for fast and dynamic layouts.
+  - title: Naming & Organizing Conventions
+    summary: |
+      From OOCSS to ITCSS, SMACSS, BEM, and atomic design,
+      there are a number of naming and organizing
+      conventions that can help us write
+      modular & maintainable CSS.
+      We'll look at what these systems have in common,
+      and how to adapt those principles for the needs
+      of your team or project.
+  - title: Design Systems & Component Libraries
+    summary: |
+      Reusable design & component systems
+      can improve consistency,
+      code-quality,
+      and team communication.
+      You don't need a dedicated team and budget
+      to get started.
+      We'll talk about planning and designing systems
+      in an agile and integrated environment.
+  - title: Style Guides & Documentation
+    summary: |
+      Documentation is central to maintainable code,
+      but how do we make sure the documentation
+      itself is maintained and up-to-date?
+      We'll look at workflow and tooling
+      for automated and integrated style guides
+      and documentation.
+  - title: Tooling & Testing
+    summary: |
+      The tools we use help define our workflow,
+      and the long-term reliability of our code.
+      Tools & tests should help encourage best-practice
+      without locking us into antiquated systems.
+      We'll talk about designing
+      the right tooling and testing workflow
+      for your team or project.
+  - title: Team Process & Workflow
+    summary: |
+      Struggling with messy handoff
+      or communication between designers & developers?
+      We can help you find the
+      practical workflows that keep your
+      process running smoothly.
+  - title: HTML & CSS in JS Frameworks
+    summary: |
+      With the advent of JavaScript frameworks,
+      we need to rethink exactly how the
+      core web languages relate and work together.
```

Comment:
It still seems odd to me to present the "advent of JavaScript frameworks" as a recent thing. Do you mean the proliferation? Or maybe "we've needed" would make it clearer that this is not a new problem?

Reply:
Still? Did you mention it before? Sorry if I missed that. Happy to consider, but heading to the airport. Propose a change?

Reply:
Yeah, no prob -- it was a comment along with my earlier suggested change, but that's easy to miss. 👍